### DISCUSSION TITLE
switch moonlight方案可以使用本服务来做公网串流嘛

### DISCUSSION THREAD

**Discussion by muyu66** [natfrp/wiki#537] [使用问题]
需求：
破解后的switch，使用swtich moonlight客户端，连接公网下的一台moonlight server PC，且此PC无公网地址，无IPV6。

switch本身可以用wifi也可以用手机热点来获取网络

[a comment by an automated account is not shown]

**Comment by CodeChenL** [comment]
frp+WireGuard即可
